app/channels/room_channel.rb

```diff
@@ -38,6 +40,12 @@ def update_pot
     end
 
     def game_play(game)
+      if game.stage == "river" && game.find_players.all? { |player| player.action >= 1}
```

Comment:
This is also reads well but it would be good to take it one step further I would pull out everything after the `&&` into its own method called `players_have_action?` then this `player.action` could be its ow method called `action_left?`.